avaframe/in3Utils/geoTrans.py

```diff
@@ -831,3 +831,87 @@ def checkOverlap(toCheckRaster, refRaster, nameToCheck, nameRef, crop=False):
             raise AssertionError(message)
 
     return toCheckRaster
+
+
+def cartToSpherical(X, Y, Z):
+    """ convert from cartesian to spherical coordinates
+
+        Parameters
+        -----------
+        X: float
+            x coordinate
+        Y: float
+            y coordinate
+        Z: float
+            z coordinate
+
+        Returns
+        ---------
+        r: float
+            radius
+        phi: float
+            azimuth angle [degrees]
+        theta: float
+            for elevation angle defined from Z-axis down [degrees]
+    """
+
+    xy = X** 2 + Y**2
+    r = np.sqrt(xy + Z**2)
+     # for elevation angle defined from Z-axis down
+    theta = np.arctan2(np.sqrt(xy), Z)
+    theta = np.degrees(theta)
+    # azimuth: 0 degree is south
+    phi = np.arctan2(X, Y)
+    phi = np.degrees(phi)
+
+    return r, phi, theta
+
+
+def rotate(locationPoints, theta, deg=True):
+    """ rotate a vector provided as start and end point with theta angle
+
+        Parameters
+        -----------
+        locationPoints: list
+            list of lists with x,y coordinate of start and end point of a line
+        theta: float
+            rotation angle of the vector from start point to end point - degree default
```

Comment:
Note rotatation direction. What happens with +theta or -theta values?

Reply:
added in counterclockwise